src/nightwatch/responsive-helpers.js

```diff
@@ -0,0 +1,42 @@
+const screenWidth = (browser) => {
+  const breakpoint = browser.currentTest.name.match(/\[@(.*?)\]/)[1];
+  return browser.globals.breakpoints[breakpoint][0];
+};
+
+const resize = breakpoint => (browser, done) => {
+  const screenSize = browser.globals.breakpoints[breakpoint];
+
+  if (!screenSize) {
+    browser.end(done);
+    throw new Error(`${breakpoint} is not defined`);
+  }
+
+  browser.url(browser.launchUrl);
+  browser.resizeWindow(screenSize[0], screenSize[1], done);
+};
+
+const resizeTo = (breakpoints, suite) => {
+  const newTests = {};
+  breakpoints.forEach((breakpoint) => {
+    newTests[`Adjust Broswer to [@${breakpoint}]`] = () => resize(breakpoint);
+    Object.keys(suite).reduce((tests, key) => {
+      const resizedTests = {};
+      const value = suite[key];
+      if (typeof value === 'function') {
+        resizedTests[`${key} [@${breakpoint}]`] = value;
```

Comment:
I wonder if a slightly more elegant approach is to do a resize as part of the first test rather than have a separate test to do resizes.